stencil-library/src/components/form/form-control-text.tsx

```diff
@@ -0,0 +1,33 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'form-control-text',
+  styleUrl: 'form-control-text.scss',
+  shadow: true,
+})
+export class TextInput {
+  @Prop() label: string;
+  @Prop() name: any;
+  @Prop() onInput: (e: any) => void;
+  @Prop() error: string;
+  @Prop() value: string;
+
+  render() {
+    return (
+      <Host exportparts="label,input,input-invalid">
+        <label part="label" class="form-label">
+          {this.label}
+        </label>
+        <input
```

Comment:
suggestion/question: 
It would be more accessible if we move the `input` inside the `label`. It would help screen readers know to which field the input is associated.  Would it still work fine with that change?

Reply:
Thats a good catch, I think the label is meant to have a `for` and the input an `id` to let screen readers know the relationship between labels and inputs

Reply:
Perfect! That will be just fine.